app/models/competition.rb

```diff
@@ -563,6 +563,7 @@ def registration_full_message
 
   def reg_warnings
     warnings = {}
+    warnings[:uses_v2] = I18n.t('competitions.messages.uses_v2_registrations') if uses_new_registration_service?
```

Comment:
More nitpick: This should also share the same name with the field.
I think it's not technically a strict requirement, but certainly nice for other developers. Otherwise, V2 could be anything, perhaps the V2 database or V2 schedule editor.